atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -405,7 +407,6 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
         TransactionManager instrumentedTransactionManager =
                 AtlasDbMetrics.instrument(metricsManager.getRegistry(), TransactionManager.class, transactionManager);
 
-        instrumentedTransactionManager.registerClosingCallback(lockAndTimestampServices.close());
```

Comment:
I don't think this is something we can drop entirely; this ensures that the lock and timestamp services are closed _when the transaction manager is closed_, as compared to `closeables` in this method which seems to be things that are closed _if initialization fails_.